e2e/process-services-cloud/start-task-form-cloud.e2e.ts

```diff
@@ -208,4 +260,289 @@ describe('Start Task Form', () => {
         });
 
     });
+
+    describe('Attach content to process-cloud task form using upload widget', async () => {
+
+        beforeEach(async (done) => {
+            navigationBarPage.navigateToProcessServicesCloudPage();
+            appListCloudComponent.checkApsContainer();
+            appListCloudComponent.checkAppIsDisplayed(candidateBaseApp);
+            appListCloudComponent.goToApp(candidateBaseApp);
+            processCloudDemoPage.openNewProcessForm();
+            done();
+        });
+
+        it('[C310358] Should be able to attach a file to a form from local', async () => {
+            startProcessPage.clearField(startProcessPage.processNameInput);
+            startProcessPage.enterProcessName(uploadLocalFileProcess);
+            startProcessPage.selectFromProcessDropdown('uploadFileProcess');
+            startProcessPage.clickStartProcessButton();
+            processCloudDemoPage.clickOnProcessFilters();
+            processCloudDemoPage.runningProcessesFilter().clickProcessFilter();
+            expect(processCloudDemoPage.getActiveFilterName()).toBe('Running Processes');
+            processCloudDemoPage.editProcessFilterCloudComponent().clickCustomiseFilterHeader().setProperty('processName', uploadLocalFileProcess);
+            processCloudDemoPage.processListCloudComponent().getDataTable().waitTillContentLoaded();
+            processCloudDemoPage.processListCloudComponent().checkContentIsDisplayedByName(uploadLocalFileProcess);
+
+            processCloudDemoPage.processListCloudComponent().getDataTable().selectRow('Name', uploadLocalFileProcess);
+            processList.push(await processHeaderCloud.getId());
+            processDetailsCloudDemoPage.checkTaskIsDisplayed('UploadFileTask');
+            processDetailsCloudDemoPage.selectProcessTaskByName('UploadFileTask');
+            taskFormCloudComponent.clickClaimButton();
+
+            const localFileWidget = widget.attachFileWidgetCloud('Attachlocalfile');
+            browser.sleep(5000);
```

Comment:
Instead of browser.sleep you can use wait for form to load again.

Reply:
Can you be more specific pls? I tired few things, but no working solution here.